yb-voyager/src/query/queryparser/ddl_processor.go

```diff
@@ -1021,6 +1021,7 @@ func (mv *FunctionProcessor) Process(parseTree *pg_query.ParseResult) (DDLObject
 		SchemaName: funcSchemaName,
 		FuncName:   funcName,
 		ReturnType: GetReturnTypeOfFunc(parseTree),
+		HasSqlBody: funcNode.CreateFunctionStmt.GetSqlBody() != nil,
```

Comment:
what if `funcNode.CreateFunctionStmt` is nil? Is that possible here?

Reply:
No, that is not possible if its function node, create function stmt can't be nil.